scripts/superchain.py

```diff
@@ -205,13 +205,12 @@ def to_nethermind_chainspec(chain_name, l1, superchain, chain, genesis):
             "opGraniteTransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "granite_time"]))),
             "opHoloceneTransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "holocene_time"]))),
             "opIsthmusTransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
+            "eip7702TransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
             "eip2537TransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
             "eip2935TransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
-            "eip6110TransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
-            "eip7002TransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
-            "eip7251TransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
-            "eip7702TransitionTimestamp": fmap(hex, (lookup(config, ["hardforks", "isthmus_time"]))),
```

Comment:
EIP 7002 and 7251 are not enabled on OP after Isthmus: https://github.com/ethereum-optimism/specs/blob/main/specs/protocol/isthmus/overview.md